common/models/product.json

```diff
@@ -19,8 +19,7 @@
       "url": "string"
     },
     "document": {
-      "apis": 
-        {},
+      "apis": "object",
```

Comment:
The docs aren't 100% clear in either direction, though, so I'm not sure. @ritch would not supporting `"someProperty": { }` be considered a breaking API change, or was it never technically allowed?